spec/models/lionpath/lionpath_export_payload_spec.rb

```diff
@@ -0,0 +1,179 @@
+require 'model_spec_helper'
+
+RSpec.describe Lionpath::LionpathExportPayload do
+  subject(:export_payload) { described_class.new(submission) }
+
+  let(:status_behavior) { instance_double('StatusGiver') }
+  let(:approval_status_behavior) { instance_double('ApprovalStatus') }
+  let(:submission) do
+    instance_double('Submission',
```

Comment:
I just learned about RSPEC instance_double thanks to this code review; seems useful!